### PR TITLE
13.0 mig hotel

### PR DESCRIPTION
**Pull request by jcchoquet**
Hotel Management
Module for Complete Hotel management which includes the basic operations at a hotel/apartment.

You can manage:

- Configure Property
- Hotel Configuration
- Manage history of Check In, Check out
- Manage Folio
- Payment
- Multi company

### CODE REVIEW

File: hotel/README.rst [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+* Serpent Consulting                Services Pvt. Ltd.
+* OpenERP SA
+
+Contributors
+~~~~~~~~~~~~
+
+* Serpent Consulting Services Pvt. Ltd. <support@serpentcs.com>
+* Nikul Chaudhary <nikul.chaudhary.serpentcs@gmail.com>
+
```
Comment (SerpentCS) [inline hotel/README.rst]: Add your name in contributor

Comment (jcchoquet) [reply hotel/README.rst]: it's done.

**Comment by jcchoquet** [comment]
Hi, i have corrected all the code, but the continuous-integration is failed again, can you help me ? or force this integration ?

Regards,

Jean-Christophe CHOQUET
[Oxilia-info]
Intégration solutions d’entreprise
06.03.94.32.01 / 02.79.79.00.29
jcchoquet@oxilia-info.fr<mailto:jcchoquet@oxilia-info.fr>
www.oxilia-info.fr<https://oxilia-info.fr/>


De : SerpentCS [mailto:notifications@github.com]
Envoyé : mercredi 13 mai 2020 09:39
À : OCA/vertical-hotel <vertical-hotel@noreply.github.com>
Cc : Jean-Christophe Choquet <jcchoquet@oxilia-info.fr>; Author <author@noreply.github.com>
Objet : Re: [OCA/vertical-hotel] 13.0 mig hotel (#91)


@SerpentCS commented on this pull request.

________________________________

In hotel/README.rst<https://github.com/OCA/vertical-hotel/pull/91#discussion_r424233025>:

> +

+Credits

+=======

+

+Authors

+~~~~~~~

+

+* Serpent Consulting                Services Pvt. Ltd.

+* OpenERP SA

+

+Contributors

+~~~~~~~~~~~~

+

+* Serpent Consulting Services Pvt. Ltd. <support@serpentcs.com<mailto:support@serpentcs.com>>

+* Nikul Chaudhary <nikul.chaudhary.serpentcs@gmail.com<mailto:nikul.chaudhary.serpentcs@gmail.com>>

+

Add your name in contributor

**Comment by robinkeunen** [comment]
In order to keep the 11.0-12.0-13.0 branches. aligned, I'd suggest adding pre-commit to the branch. You can see an example here: https://github.com/OCA/vertical-hotel/pull/101/commits/2dfa468744a8a9a8964d3f14ed24f1dc16eda2f0

**Pull request closed without merging**